plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/HostedMachineProviderImpl.java

```diff
@@ -210,7 +211,7 @@ protected void pullImage(CheServiceImpl service,
                                               .withBuildArgs(buildArgs),
                               progressMonitor);
 
-        } catch (ImageNotFoundException e) {
+        } catch (DockerException e) {
```

Comment:
Pull is used in case snapshot is not used for machine. This means that this change will affect no-snapshot case. 
Do you think that in this case any internal error should be treated as image not found error?